spec/lib/administrate/search_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "spec_helper"
+require "support/constant_helpers"
+require "administrate/fields/string"
+require "administrate/fields/email"
+require "administrate/fields/number"
+require "administrate/search"
+
+class MockDashboardClass
+  ATTRIBUTE_TYPES = {
+    name: Administrate::Field::String,
+    email: Administrate::Field::Email,
+    phone: Administrate::Field::Number,
+  }
+end
+
+class MockResolver
```

Comment:
100% on board with that. I typically use Mocha as my mocking/stubbing gem of choice so when the `mock` method wasn't available, I went for a dummy class instead. I'll update this to use a double.